blackbox/docs/configuration.txt

```diff
@@ -1851,6 +1851,25 @@ to your config. This will prevent allocating primary and replica
 of the same shard on the same machine even if more than one instances
 running on it.
 
+Other System Configuration
+==========================
+
+Virtual Memory
+--------------
+
+CrateDB uses a hybrid ``mmapfs``/``niofs`` directory by default to store its
+indices. The defaultoperating system limits on ``mmap`` counts is likely to be
+too low, which may result in out of memory exceptions.
```

Comment:
that might be a good idea! perhaps one of the core engineers knows more?

also from the snippet above I'm spotting a missing space in "defaultoperating"